images/diarizer/model_download.py

```diff
@@ -0,0 +1,4 @@
+from pyannote.audio import Pipeline
+pipeline = Pipeline.from_pretrained(
+    "pyannote/speaker-diarization-3.1",
+    use_auth_token="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")
```

Comment:
@ajbouh this seems to do the trick by populating the cache so that they're loaded from there instead of re-downloading (verified by removing the access token from `app.py`).
However, it does increase the size of the Docker image with the model files. 
I guess this way I'm also not sure there's a way to pin each of the recursive dependencies.

With the resourcedirs `#imagespec` approach I guess there's not a way to specify multiple commands? I suppose we could make another wrapper container for the huggingface-cli that accepts multiple repos to download. That seems like their recommendation is to download the repos you need into the cache and load from there instead of the `--local-file` option:
https://huggingface.co/docs/huggingface_hub/en/guides/cli#specify-cache-directory

Or you mentioned mounting a union of the models. Is there a simpler way to to that, or I guess if I know the cache folder from each resource dir, I could specify the individual mount points to make a folder with all of the cache contents.